ballerina/Module.md

```diff
@@ -0,0 +1,101 @@
+## Overview
+
+[Guidewire InsuranceNow](https://www.guidewire.com/products/insurancenow) is a cloud-based insurance platform offering comprehensive tools for policy, billing, and claims management, designed to streamline operations and improve customer service in the insurance industry.
+
+The `ballerinax/guidewire.insnow` package provides a client API to connect to Guidewire InsuranceNow cloud API using Ballerina.
+
+## Compatibility
+
+|                        |   Version    |
+|:-----------------------|:------------:|
+| Guidewire InsuranceNow | **2023.3 (Innsbruck)** |
+
+## Setup guide
+
+To use the Ballerina Guidewire InsuranceNow connector, access to the Guidewire API is required. Access is granted to partners or customers of the Guidewire platform. To become a partner or customer, reach out to Guidewire's sales team via the [Guidewire website](https://www.guidewire.com/contact-us/). For detailed instructions on obtaining authentication/authorization keys, please go through the [portal development guide](https://docs.guidewire.com/cloud/in/20233/portaldev/PortalDevelopment/topics/c_overview_portal_development.html).
```

Comment:
Currently we don't have access to an guidewire account. Hence, we could not add the screenshots for the setup guide.